doc/api/stream.md

```diff
@@ -443,9 +443,26 @@ first argument. To reliably detect write errors, add a listener for the
 The return value is `true` if the internal buffer does not exceed
 `highWaterMark` configured when the stream was created after admitting `chunk`.
 If `false` is returned, further attempts to write data to the stream should
-stop until the [`'drain'`][] event is emitted. However, the `false` return
-value is only advisory and the writable stream will unconditionally accept and
-buffer `chunk` even if it has not not been allowed to drain.
+stop until the [`'drain'`][] event is emitted.
+
+In case the `false` return value is ignored, the writable stream will
+unconditionally accept and buffer `chunk` even if it has not not been allowed
+to drain. However, this might lead to memory leaks and garbage
+collection issues, especially if the writable is a [Transform]()
```

Comment:
Should it be `[Transform][]`? for linking?

This may be more readable if like:

`especially if the writable stream is a [Transform]()` or even `especially if the stream is a [Transform]()`?